src/OrchardCore/OrchardCore/OrchardCore.csproj

```diff
@@ -14,6 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="AsyncKeyedLock" Version="6.0.5" />
```

Comment:
Move the version to `Dependencies.props`

Reply:
Done. No need for Dependencies.AspNetCore.props right?